src/Sylius/Behat/Page/Admin/Product/ShowPage.php

```diff
@@ -32,10 +33,16 @@ public function getRouteName(): string
         return 'sylius_admin_product_show';
     }
 
+    public function specifyChannel(ChannelInterface $channel): void
+    {
+        $this->getElement('scrolling_menu')->find('css', sprintf("a:contains('%s')", $channel->getName()))->click();
```

Comment:
`->find('css', sprintf("a:contains('%s')", $channel->getName()))->click()` can be replaced with `->clickLink($channel->getName())`